Wikipedia/Code/LanguagesViewController.m

```diff
@@ -103,40 +103,16 @@ - (void)loadLanguages {
 }
 
 - (void)downloadArticlelanguages {
-    [[WMFAlertManager sharedInstance] showAlert:MWLocalizedString(@"article-languages-downloading", nil) sticky:YES dismissPreviousAlerts:NO tapCallBack:NULL];
-    // (temporarily?) hide search field while loading languages since the default alert UI covers the search field
-    [self setLanguageFilterHidden:YES animated:NO];
-
     @weakify(self);
     [self.titleLanguageController
      fetchLanguagesWithSuccess:^{
         @strongify(self)
-        //This can fire rather quickly, lets give the user a chance to read the message before we dismiss
-        dispatchOnMainQueueAfterDelayInSeconds(1.0, ^{
-            [[WMFAlertManager sharedInstance] dismissAlert];
-        });
-        [self setLanguageFilterHidden:NO animated:YES];
         [self reloadDataSections];
     } failure:^(NSError* __nonnull error) {
         [[WMFAlertManager sharedInstance] showErrorAlert:error sticky:YES dismissPreviousAlerts:YES tapCallBack:NULL];
     }];
 }
 
-#pragma mark - Search Bar Visibility
-
-- (void)setLanguageFilterHidden:(BOOL)hidden animated:(BOOL)animated {
```

Comment:
@coreyfloyd I don't feel super strongly about removing the alert, but it was easy and @nirzar asked me to.